frontend/components/dynamic-badge-maker.js

```diff
@@ -64,7 +64,7 @@ export default class DynamicBadgeMaker extends React.Component {
           className="short"
           value={this.state.color}
           onChange={event => this.setState({ color: event.target.value })}
-          placeholder="hex color" /> {}
+          placeholder="color" /> {}
```

Comment:
Would it be helpful to add some examples here, or else link to elsewhere in the docs where the allowed values are enumerated?

Reply:
Not quite sure where it would go?
I've added some info [here](https://github.com/badges/shields/pull/1635/files#diff-77b808a4aca9e41befff69ef8d77a0beR193)
```diff
-<td>Set background of the left part (hex color only)</td>
+<td>Set background of the left part (hex, rgb, rgba, hsl, hsla and css named colors supported)</td>
```

Reply:
That seems good to start!